includes/settings/js/src/components/Tools.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { __ } from "@wordpress/i18n";
+import ExportFileButton from "./ExportFileButton";
+const { wp } = window;
+const { apiFetch } = wp;
+
+export default function Tools() {
+	/**
+	 * Gets model export data via the REST API.
+	 */
+	function getModels() {
+		return apiFetch({
+			path: `/wpe/atlas/content-models/`,
+			method: "GET",
+			_wpnonce: wpApiSettings.nonce,
+		});
+	}
+
+	/**
+	 * Format filename for export
+	 * @returns {string}
+	 */
+	function getFormattedDateTime() {
+		var today = new Date();
+		var y = today.getFullYear();
+		var m = today.getMonth() + 1;
+		var d = today.getDate();
+		var h = today.getHours();
+		var mi = today.getMinutes();
+		var s = today.getSeconds();
+		return m + "-" + d + "-" + y + "-" + h + "-" + mi + "-" + s;
+	}
+
+	return (
+		<div className="app-card tools-view">
+			<section className="heading flex-wrap d-flex flex-column d-sm-flex flex-sm-row">
+				<h2>{__("Tools", "atlas-content-modeler")}</h2>
+			</section>
+			<section className="card-content">
+				<div className="row">
+					<div className="col-xs-10 col-lg-4 order-1 order-lg-0">
+						<div className="row">
+							<div className="col-xs-12">
+								<h4>
+									{__(
+										"Import Model",
```

Comment:
```suggestion
										"Import Models",
```

I know the mocks show singular, but it's more accurate to use plural.